src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -106,6 +108,7 @@ public void move(double power) {
     }
 
```

Comment:
A suggestion, use in TrigonDrive this log:
[https://oblarg.github.io/Oblog/io/github/oblarg/oblog/annotations/Log.DifferentialDrive.html](url)